### DISCUSSION TITLE
OAuth on behalf of a customer?

### DISCUSSION THREAD

**Discussion by tzusman** [etsy/open-api#764] [Q&A] [answered]
Is it possible to have a customer connect via OAuth and then use the API to retrieve order details, including tracking numbers for shipping?

It appears the v3 API is entirely shop centric, but maybe I'm missing something.

**Comment by etsyachristensen** [comment, marked as answer]
Unfortunately no. The API is designed for shop owners and their 3rd party integrations.

**Reply by tzusman** [reply]
got it, thanks
